exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatchSizer.java

```diff
@@ -50,7 +50,7 @@
 public class RecordBatchSizer {
   private static final int OFFSET_VECTOR_WIDTH = UInt4Vector.VALUE_WIDTH;
   private static final int BIT_VECTOR_WIDTH = UInt1Vector.VALUE_WIDTH;
-  private static final int STD_REPETITION_FACTOR = 10;
+  public static final int STD_REPETITION_FACTOR = 10;
```

Comment:
This is another of those silly fudge factors that really have no meaning. The value of 10 came from the vector allocation code in `AllocationHelper` (or I thought it did, the magic number there is 5.)

Maybe move this to `AllocationHelper` and set it to 5, then use it here and in `AllocationHelper` so we use a consistent guess everywhere.

Reply:
done. using 5 in both places now.